apps/pkcs12.c

```diff
@@ -211,7 +208,7 @@ int pkcs12_main(int argc, char **argv)
             macver = 0;
             break;
         case OPT_DESCERT:
-            cert_pbe = NID_pbe_WithSHA1And3_Key_TripleDES_CBC;
+            cert_pbe = NID_aes_256_cbc;
```

Comment:
How is this related to des?

Reply:
IMO it should either fail OR
use des and set use_legacy

Reply:
@slontis Thanks for your comment. The latter would reset key_pbe as well. Maybe we don't want that. So I could just set it back to - cert_pbe = NID_pbe_WithSHA1And3_Key_TripleDES_CBC here and nothing else. What do you think?

Reply:
IMO 3DES (at least for now) should work without legacy provider, shouldn't it? So yes, just dropping this change should suffice.

Reply:
It would be worth explicitly documenting that -descert will use 3DES for the cert encryption and AES will be used for key unless -legacy is used. Also use_legacy handling needs to be adjusted to not override the cert_pbe if it is set to  NID_pbe_WithSHA1And3_Key_TripleDES_CBC already.

Reply:
> IMO 3DES (at least for now) should work without legacy provider, shouldn't it? So yes, just dropping this change should suffice.

yes it works without legacy provider.